virt/virtual_machines/virtual_disks/virt-cloning-a-datavolume-using-smart-cloning.adoc

```diff
@@ -22,3 +22,5 @@ include::modules/virt-cloning-a-datavolume.adoc[leveloffset=+1]
 * xref:../../../virt/virtual_machines/cloning_vms/virt-cloning-vm-disk-into-new-datavolume.adoc#virt-cloning-pvc-of-vm-disk-into-new-datavolume_virt-cloning-vm-disk-into-new-datavolume[Cloning the persistent volume claim of a virtual machine disk into a new data volume]
 
 * xref:../../../virt/virtual_machines/virtual_disks/virt-using-preallocation-for-datavolumes.adoc#virt-using-preallocation-for-datavolumes[Configure preallocation mode] to improve write performance for data volume operations.
+
```

Comment:
The link is good, but there are two things that can be improved:
1. line 13: "if you storage provider supports smart-cloning." should be "if **your** storage provider supports smart-cloning."

2. in "modules/virt-cloning-a-datavolume.adoc"
There is one more Prerequisite: 
"The source and target PVCs share the same Volume Mode"

Reply:
Thanks @brybacki and sorry for the embarrassing typos :-(

Reply:
;)